scl/slack/slack.conf

```diff
@@ -0,0 +1,65 @@
+#############################################################################
+# Copyright (c) 2018 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+@requires http
+@requires add-contextual-data
+
+# Throttle: https://api.slack.com/docs/rate-limits
+block destination slack(
+  hook-url()
+  author-template('"host: ${HOST} | program: ${PROGRAM}(${PID}) | severity: ${PRIORITY}"')
```

Comment:
Why do we call this author-template? Does it refer to some slack terminology? Can we simply call it `template`?

Reply:
Yes, slack has an author field, which I wanted to be modifiable. On default this will print the HOST, PROGRAM, PID and SEVERITY values, but one can change it however they like.

Reply:
template option is already used to the body (=text payload)